File: 1210-mean-of-array-after-removing-some-elements/1210-mean-of-array-after-removing-some-elements.java
class Solution {
    public double trimMean(int[] arr) {
        int removeLen = arr.length / 20;
        Arrays.sort(arr);
        double sum = 0.0;
        for(int i = removeLen; i < arr.length - removeLen; i++){
            sum += arr[i];
        }
        double mean = sum / (arr.length - 2 * removeLen);
        return mean;
    }
}

File: 1210-mean-of-array-after-removing-some-elements/1210-mean-of-array-after-removing-some-elements.py
class Solution:
    def trimMean(self, arr: List[int]) -> float:
        l = len(arr)
        remove_len = l // 20
        arr.sort()
        s = sum(arr[remove_len:l - remove_len])
        return s / (l - 2 * remove_len)